tests/accessor_legacy/accessor_api_buffer_common.h

```diff
@@ -732,18 +729,16 @@ class check_buffer_accessor_api {
     // and a write to dataMultiDimSyntax
     static constexpr bool useIndexesWrite = (dims > 0);
     auto dataIdSyntax = get_buffer_input_data<T>(count, dims);
-    auto dataMultiDimSyntax =
-        get_buffer_input_data<T>(count, dims, useIndexesWrite);
+    auto dataMultiDimSyntax = get_buffer_input_data<T>(count, dims);
 
-    static constexpr bool isHostBuffer =
-        (target == sycl::target::host_buffer);
-    auto errors = get_error_data(isHostBuffer ? 2 : 4);
+    static constexpr bool isHostBuffer = (target == sycl::target::host_buffer);
 
+    auto errors = get_error_data(isHostBuffer ? 2 : 4);
```

Comment:
`isHostBuffer` ? 2 : 4` is a magical incantation used twice.
Put in in a variable with a clear good name which will be used twice.

Reply:
Done